app/javascript/packages/react-hooks/README.md

```diff
@@ -0,0 +1,3 @@
+# `@18f/identity-react-hooks`
+
+A collection of general-purpose [React hooks](https://reactjs.org/docs/hooks-intro.html).
```

Comment:
Is it worth documenting the hooks in the README with examples? Or realistically these are internal so we should just look for internal usage?